src/clients/http_client/http_client.ts

```diff
@@ -211,18 +211,18 @@ class HttpClient {
             case response.status === StatusCode.TooManyRequests: {
               const retryAfter = response.headers.get('Retry-After');
               throw new ShopifyErrors.HttpThrottlingError(
-                `Shopify is throttling requests${errorMessage}`,
-                retryAfter ? parseFloat(retryAfter) : undefined,
-              );
+              `Shopify is throttling requests${errorMessage}`,
+              retryAfter ? parseFloat(retryAfter) : undefined,
+            );
```

Comment:
This indentation here feels very counter-intuitive to me. Is it part of the new rules?

Reply:
ohh I was trying out an indentation rule and used the `--fix` flag. I'll revert these changes because they were better before